sig-cloud-provider/CHARTER.md

```diff
@@ -11,27 +11,32 @@ SIG Cloud Provider’s mission is to simplify, develop, and maintain cloud provi
 
 #### Areas of Focus
 
-- Extension points between Kubernetes and any cloud provider
+- Cloud provider specific integrations and extension points that are not already covered by a more specific sig such as storage or networking.
 - APIs/interfaces for efficiently provisioning/de-provisioning cloud resources (nodes, routes, load balancers, etc)
 - Configuration of cluster components to enable cloud provider integrations
 - Testing and testing frameworks to ensure vendor neutrality across all cloud providers
 
 #### Code, Binaries and Services
 
+The SIG offers standardization across cloud-provider-* repos that are owned by the sig. We establish basic structure and tooling expectations to help new contributors to understand the code and how to contribute.
+
 - the [common interfaces](https://github.com/kubernetes/cloud-provider/blob/master/cloud.go) consumed by all cloud providers
 - the [cloud-controller-manager](https://github.com/kubernetes/kubernetes/tree/master/cmd/cloud-controller-manager), which acts as the “out-of-tree” cloud provider component for clusters.
 - core controllers (started by the cloud-controller-manager) that interact with cloud provider resources
 - all [cloud provider repositories](https://github.com/kubernetes?utf8=%E2%9C%93&q=cloud-provider-&type=&language=) under the Kubernetes organization
 - [e2e tests for cloud provider specific](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/cloud) functionality
+- the subproject [apiserver-network-proxy](https://github.com/kubernetes-sigs/apiserver-network-proxy), which is an extensible system which controls network traffic from the Kube API Server.
 - all the subprojects formerly owned by [SIG-AWS](https://github.com/kubernetes/community/tree/master/sig-aws#subprojects), [SIG-AZURE](https://github.com/kubernetes/community/tree/master/sig-azure#subprojects), [SIG-GCP](https://github.com/kubernetes/community/tree/master/sig-gcp#subprojects), [SIG-IBMCloud](https://github.com/kubernetes/community/tree/master/sig-ibmcloud#subprojects), [SIG-Openstack](https://github.com/kubernetes/community/tree/master/sig-openstack#subprojects), [SIG-VMware](https://github.com/kubernetes/community/tree/master/sig-vmware#subprojects).
 - any new subproject that is cloud provider specific, unless there is another SIG already sponsoring it.
 
 #### Cross-cutting and Externally Facing Processes
```

Comment:
i think it would be nice to add a note about our participation in cross-sig working groups, like the node lifecycle working group.